main/src/cgeo/geocaching/maps/LivemapStrategy.java

```diff
@@ -9,7 +9,6 @@
  * Defines the strategy for the Live Map
  */
 public enum LivemapStrategy {
-    FASTEST(1, EnumSet.of(Flag.LOAD_TILES), R.string.map_strategy_fastest),
```

Comment:
Maybe a note explaining why we don't (and can't) have a value of 1 anymore would be useful.